app/views/users/index.html.erb

```diff
@@ -1 +1,7 @@
-<h1>Hi there</h1>
+<div class="all-users-container">
+<% @users.each do |user| %>
+    <%= link_to(user) do %>
+      <%= render "shared/user_details", user: user %>
```

Comment:
- [ ] @Ibtisam34 kindly check the rendered file it doesn't exist in your shared folder ,kindly rectify to solve the error below 
![image](https://user-images.githubusercontent.com/61408860/225052898-ae6a18f1-1077-4ee8-a7f3-a53119b450dd.png)